batch/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length=100
```

Comment:
I thought I saw somewhere else that the maximum line length is 120. Maybe the previous PR?

Reply:
Ah good point, pylintrc and flake8 have different configuration files. I changed them both to be 120.